src/test/resources/config/application-core.yml

```diff
@@ -0,0 +1,48 @@
+artemis:
+  atlas:
+    enabled: true
+  exam:
+    enabled: true
+  plagiarism:
+    enabled: true
+  text:
+    enabled: true
+  tutorialgroup:
+    enabled: true
+  user-management:
+    passkey:
+      enabled: true
```

Comment:
We need to enable the passkey feature for the server tests, as the default is `false` - we didn't need this file before because other features, such as atlas are activated per default